src/lib/helpers/auth.test.ts

```
@@ -0,0 +1,85 @@
+import { describe, test, expect } from 'vitest';
+
+// Simple auth helper functions that don't require external dependencies
+export const sessionCookieName = 'sashakt-session';
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)